tests/testthat/test-augsynth_plotting.R

```diff
@@ -0,0 +1,98 @@
+library( tidyverse )
+data(basque, package = "Synth")
+
+context("Test plotting features of single augsynth objects")
+
+basque <- basque %>% mutate(trt = case_when(year < 1975 ~ 0,
+                                            regionno != 17 ~ 0,
+                                            regionno == 17 ~ 1)) %>%
+    filter(regionno != 1)
+
+syn <- augsynth(gdpcap ~ trt, regionno, year, basque, progfunc = "Ridge", scm = T)
+
+test_that("All plot types for plot.summary.augsynth() are working", {
```

Comment:
This is a super large test and it isn't really helpful because if it errors then we don't know what caused the error. And all it tests is that the plots actually are produced. I think this can safely be dropped, or is there a more concise test that would be useful?